### ISSUE TITLE
[Feature] Support Intel ARC GPUs

### ISSUE THREAD

**Issue by Fredrik81** [MauriceNino/dashdot#1067]
### Description of the feature

Support for Intel ARC GPU's would be nice.
Doesn't seem to be supported at least with the ":latest" docker image.

**Comment by holzerseb** [comment]
Would also very much appreciate support for intel gpus, thanks!

**Comment by Hans03430** [comment]
Are the intel Xe IGPUs soported by default?

**Comment by MauriceNino** [comment]
Yes, this is currently not supported, but I can't really do much here, as I have no Intel GPU. Would be open for a PR for a separate image, similar to the NVIDIA one.

**Comment by KeesTucker** [comment]
try `image: ghcr.io/keestucker/dashdot:intel` I have opened a PR.
